src/components/Card.js

```diff
@@ -29,12 +71,19 @@ export default class Card {
     this._deleteButton = this._cardElement.querySelector(
       ".card__delete-button"
     );
+    this._deleteCardModal = document.querySelector(".delete-card-modal");

```

Comment:
Please, delete it

* Any modal should be created using a class, which extends `Popup` (it handles the overlay)